### ISSUE TITLE
text/value binding does not work for option element in foreach in IE8

### ISSUE THREAD

**Issue by nexus-uw** [knockout/knockout#1825]
Hello,

I am trying to generate a select element with option groups. My code has worked without issue on modern browsers, but it breaks in IE8.

The following example shows what I am trying to do:

```html
<head>
      <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-shim.min.js"></script>
</head>
<body>
  <select data-bind="foreach:groups">
    <optgroup data-bind="foreach:options, attr : { label: name }">
      <option data-bind="text:text, value:text">
      </option>
    </optgroup>
  </select>
 <script src="//cdnjs.cloudflare.com/ajax/libs/knockout/3.3.0/knockout-debug.js"></script>
 <script type="text/javascript">
 var test = {
   groups : [{
     name : 'name 1',
     options :[{text:'opt11'}, {text:'opt12'}]
   },
   {
     name : 'name 2',
     options :[{text:'opt21'}, {text:'opt22'}]
   }]
 };
 ko.applyBindings(test);

 </script>
</body>
```

the above 

results in the following in IE8
```
'text' is undefined
```

changing ```data-bind="text:text, value:text"``` to ```data-bind="text:$data.text, value:$data.text"``` will cause the following to occur (again in IE8)
```
Unexpected call to method or property access.
```

finally dropping the text binding will allow IE8 to not throw an error but the select box is not rendered correctly (it is just blank with no values)
![value only binding](https://cloud.githubusercontent.com/assets/3188890/8556488/4c287c2a-24c5-11e5-84a3-c3b57e8c8d55.png)

Any suggestions on how to fix this would be greatly appreciated.

**Comment by nexus-uw** [comment]
Update: including jquery 1.11.... fixed this issue on IE8.

It would be nice to know why pulling jQuery in would fix this issue.

**Comment by mbest** [comment]
When jQuery is included, Knockout will use it to parse and copy HTML templates in IE8. Otherwise Knockout has a much simpler method for doing so, which obviously fails for `<select>`.

**Comment by mbest** [comment]
Related: #1773

**Comment by mbest** [comment]
Fixed with commit 4c19469.
